portfolio/src/main/webapp/script.js

```diff
@@ -13,18 +13,21 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Fetches comments stored in backend servlet.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function getComments() {
+  fetch('/data')
+    .then(response => response.json())
+    .then(comments => {
+      const section = document.getElementById('comment-section');
+      
+      for (comment of comments) {
```

Comment:
Please don't forget to declare variables. And add `"use strict";` at the top of the file to prevent this.
Now, `comment` is a global variable (`window.comment`)

Reply:
Done